lib/dams/random_controller_helper.rb

```diff
@@ -1,9 +1,9 @@
 module Dams
   module RandomControllerHelper
     # Retrieve random page from solr index
-    def random_page(q)
-      @response, @document = get_search_results(:q => "#{q}", :rows => 1)
-      page_count = @response.response['numFound']/20.to_f   
+    def random_page(q, fq)
+      @response, @document = get_search_results(:qf => "#{q}", :fq => "#{fq}", :rows => 100)
```

Comment:
@mcritchlow You're right, I forgot to change it back to 1 when doing testing.  
For that function, we just need to get the number of results found.  I will change it back to 1.